recipes/pySW4/meta.yaml

```diff
@@ -17,6 +17,7 @@ build:
 
 requirements:
   build:
+    - noarch: python
```

Comment:
Sorry, wasn't clear before: `noarch: python` should be under the top-level `build` section above, not the `requirements/build` section here. It's metadata about the build, not a conda package dependency. :)